docs/topics/web-create-project.adoc

```diff
@@ -76,7 +76,7 @@ This option should not be used if some classes are unavailable for analysis.
 * *Exploded app*: The input directory contains the unpackaged source files of an application.
 * *Keep work dirs*: Retains temporary files, for example, the graph database or extracted archive files, for debugging purposes.
 * *Skip reports*: HTML reports are not generated. Must be enabled if you enabled *Export CSV*.
-* *Allow network access*: This option reduces performance.
+* *Allow network access*: Validates any XML files within the analyzed applications against their schema. This option may reduce performance.
```

Comment:
@RichardHoch 
We are not supposed to use 'may' for indicating possibility. Use 'might' instead.
Also, to be consistent within this list of options, consider making this warning into a note:
+
[NOTE]
====
This option might reduce performance.
====